assets/app/actions/actionCreators/alertActions.js

```diff
@@ -0,0 +1,39 @@
+import { alertActionTypes as types } from '../../constants';
+
+const authErrorType = types.AUTH_ERROR;
```

Comment:
I like having constants located where they make sense and matter instead of extracted to some "always include me as a dependency" file. This is a step along the path toward moving constants to the code that is coupled with their definition.

I also like exporting the types as we can eventually write a test to ensure no two action types have a name collision.

Our tests should verify the actionCreator is well formed and meets our expectations.